client/src/styles/settings/color.css

```diff
@@ -11,16 +11,19 @@
     --(…)dark:  darker value
 */
 
+/* If Design doc uses color on the left, then use color on the right
```

Comment:
One day (maybe) the design colors may be updated… should I add that to this comment?

Reply:
> One day (maybe) the design colors may be updated… should I add that to this comment?

@wesleyboar Add that they may change in the future? Sure, why not

Reply:
Code has newer colors. Design has older colors. I mean the design may eventually be updated to also use the new colors. So, I added:

```
(When design is updated to use the new colors, we can remove this)
```